app/webpacker/lib/requests/routes.js.erb

```diff
@@ -306,6 +306,25 @@ export const bulkUpdateRegistrationUrl = `<%= CGI.unescape(Rails.application.rou
 
 export const paymentTicketUrl = (competitionId, donationIso) => `<%= CGI.unescape(Rails.application.routes.url_helpers.api_v1_registrations_payment_ticket_path(competition_id: "${competitionId}", donation_iso: "${donationIso}")) %>`;
 
+export const rankingsUrl = (eventId, rankingType, region, gender, show) => {
+  const url = `<%= CGI.unescape(Rails.application.routes.url_helpers.rankings_path(event_id: "${eventId}", type: "${rankingType}")) %>`
+  const queryParams = new URLSearchParams();
+
+  if (region !== 'world') {
+    queryParams.append('region', region);
+  }
+
+  if (gender !== 'All') {
+    queryParams.append('gender', gender);
+  }
+
+  if (show !== '100 persons') {
+    queryParams.append('show', show);
+  }
```

Comment:
If you use this for querying the JSON, you can just always append the parameters. The controller will internally append them anyways, so if you already append them here it saves you a few if-queries for the same end result.

Reply:
I was thinking it would be slightly less annoying for people sharing links